src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$color-blue-accent: rgba(0, 172, 220, 1);
+$border-color: #f3f3f3;
+$txt-color: #060b35;
```

Comment:
Consider using a more descriptive name for the text color variable, such as $text-color or $color-text.